preprocessors/celebrity.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/celebrity-detector.schema.json",
+    "type": "object",
+    "title": "Celebrity Detector",
+    "description": "Preprocessor to determine detect celebrities in an image. The associated information can be found in preprocessor named 'ca.mcgill.a11y.image.preprocessor.celebrityDetector'. ",
+    "properties":{
+        "celebrity":{
+            "type":"array",
+            "description": "Contains a list of all celebrities",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "ID":{
+                        "description": "The ID of a person. The ID is generated by object detection preprocessor and is reused in this preprocessor",
+                        "type": "integer"
```

Comment:
Is this actually the only limit on IDs? IIRC, they are always greater than or equal to 0.